status/backend.json

```diff
@@ -14,7 +14,7 @@
       "program_active": true
     },
     "zendesk": {
```

Comment:
I guess `assistanceTool` should replace `zendesk`

Reply:
Yes, you're right.
Modified in [5c12c7f](https://github.com/pagopa/io-services-metadata/pull/525/commits/5c12c7f632ae46ef9c99cffd1949428c314a7dfc)